venues/icaps-conference.org/ICAPS/2019/Workshop/Hierarchical_Planning/python/init.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+
+import argparse
+import openreview
+from openreview import tools
+from openreview import invitations
+import config
+import datetime
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+
+	baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+ 	username - the email address of the logging in user
+	password - the user's password
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print('connecting to {0}'.format(client.baseurl))
+
+conference = config.get_conference(client)
+# March 24 midnight anywhere on earth
+conference.open_submissions(due_date = datetime.datetime(2019, 3, 25, 11, 59))
```

Comment:
I think it is 10:59.

Reply:
I disagree.  On the web it says it's 12 hours off.  Show me where it's 11 hours difference and I'll change it.  Actually I should change it to 12:00.

Reply:
You are right. I was confused to Samoa time, that is -11/